books/finance/bake

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '11.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :finance) do |doc|
+  include Kitchen::Directions
+
+  # Set overrides
+  doc.selectors.override(
+    page_summary: 'section.section-summary'
+  )
+
+  book = doc.book
+  metadata = book.metadata
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book, title_element: 'h1')
+  BakeChapterIntroductions.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+
+  BakeAutotitledNotes.v1(
+    book: book,
+    classes: %w[think-through link-to-learning concepts-practice]
+  )
+
+  BakeNumberedNotes.v1(
+    book: book,
+    classes: %w[cfa-institute]
+  )
+
+  # book_answer_key = BookAnswerKeyContainer.v1(book: book, solutions_plural: false)
+  book.chapters.each do |chapter|
+    BakePageAbstracts.v1(chapter: chapter)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata, klass: 'section-summary')
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+
+    chapter.pages.search('section').injected_questions.each do |question|
+      BakeInjectedExerciseQuestion.v1(question: question, number: question.count_in(:section))
+    end
+
+    title_snippet_sections = %w[cfa-institute sources-notes]
+    title_snippet_sections.each do |section_key|
+      MoveCustomSectionToEocContainer.v1(
+        chapter: chapter,
+        metadata_source: metadata,
+        container_key: section_key,
+        uuid_key: ".#{section_key}",
+        section_selector: "section.#{section_key}"
+      ) do |section|
+        RemoveSectionTitle.v1(section: section)
+        title = EocSectionTitleLinkSnippet.v1(page: section.ancestor(:page))
+        section.prepend(child: title)
+      end
+    end
+
+    eoc_sections = %w[multiple-choice review-questions problem-set video-activity]
+    eoc_sections.each do |section_key|
+      MoveCustomSectionToEocContainer.v1(
+        chapter: chapter,
+        metadata_source: metadata,
+        container_key: section_key,
+        uuid_key: ".#{section_key}",
+        section_selector: "section.#{section_key}"
+      ) do |section|
+        RemoveSectionTitle.v1(section: section)
+      end
+
+      chapter.search("section.#{section_key}").exercises.each do |exercise|
+        BakeNumberedExercise.v1(exercise: exercise, number: exercise.count_in(:chapter))
+
+        next unless section_key == 'multiple-choice'
+
+        BakeAccessibilityFixes.v1(section: exercise)
+      end
```

Comment:
I don't think this is doing anything anymore, because `exercises` don't exist in the content with the injected exercises update